### PR TITLE
Added Tree Traversal (preorder, inorder, post order)

### LINKED ISSUE #5: Implement Tree Traversal Algorithms
Missing: Tree Traversals (Inorder, Preorder and Postorder) implementation

more on tree traversal can be found here:https://en.wikipedia.org/wiki/Tree_traversal#Types

### PR DESCRIPTION
**Pull request by MansiAyer**
Fixes #5

### CODE REVIEW

File: Binary Tree Traversal/preorder-inorder-postorder.c [hunk 1]
```diff
@@ -0,0 +1,268 @@
+#include<stdio.h>
+#include<malloc.h>
+#include<stdlib.h>
+struct node
+{
+int data;#include<stdio.h>
```
Comment (saurabh47) [inline Binary Tree Traversal/preorder-inorder-postorder.c:6]: * Please check your code.You have added `#include<stdio.h>`.
also close properly close the `struct node`
 * Please add README.md file inside `Binary Tree Traversal` directory.write some description about algorithm & input & output sample.

**Review by saurabh47 — CHANGES_REQUESTED** [review]

**Comment by MansiAyer** [comment]
Sorry for taking so long to respond, I had a few assignments from college so I was short on time. I'll update this by tomorrow afternoon!

**Review by saurabh47 — CHANGES_REQUESTED** [review]
Nice implementation @MansiAyer with extra search functionality.
Please rename the file `ReadMe.md` to `README.md`

**Comment by MansiAyer** [comment]
Done 👍

**Review by saurabh47 — APPROVED** [review]

**Pull request merged**